README.md

```diff
@@ -20,7 +20,7 @@ Create a `.env` file in the root directory of this repository:
 
 | Variable name               | Required?              | Description                                                                                                           | Example                                               | Default |
 |-----------------------------|------------------------|-----------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|---------|
-| `GSI_QC_ETL_ROOT_DIRECTORY` | **Yes**                | Directory where the QC-ETL caches are located                                                                         | `/qcetl`                                              | |
+| `GSI_QC_ETL_ROOT_DIRECTORY` | **Yes**                | One or more colon seperated directories where the QC-ETL caches are located                                           | `/qcetl` or `/qcetl:/qcetl_archive`                    | |
```

Comment:
For the archival db-backed cache, is this the directory where the metadata is located?

Reply:
I think it would be valuable to specify here that in case of duplicate records, the records from the directory/cache listed first will be selected.

Reply:
> For the archival db-backed cache, is this the directory where the metadata is located?

Correct. The only requirement for gsiqcetl is the `config.json`, which specifies how to connect to the database.